border-radius-previewer/src/App.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+
+const App: React.FC = () => {
+  const [upperLeft, setUpperLeft] = useState<string>("0px ");
+  const [upperRight, setUpperRight] = useState<string>("0px ");
+  const [lowerLeft, setLowerLeft] = useState<string>("0px ");
+  const [lowerRight, setLowerRight] = useState<string>("0px ");
+  const [roundedCornersCss, setRoundCornersCss] = useState<string>("0px 0px 0px 0px");
+  const [backgroundColor, setBackgroundColor] = useState<string>("rgb(103, 143, 141)");
+  const [padding, setPadding] = useState<string>("100px");
+
+  function onChangeHandler(id: string, value: string): void {
+    setBorderRadiuses(id, value);
+    setTextarea();
+  }
+
+  function setBorderRadiuses(id: string, value: string): void {
+    if (id === "upperLeft") {
+      setUpperLeft(value + "px ");
+    } else if (id === "upperRight") {
+      setUpperRight(value + "px ");
+    } else if (id === "lowerLeft") {
+      setLowerLeft(value + "px ");
+    } else if (id === "lowerRight") {
+      setLowerRight(value + "px ");
+    }
+  }
+
+  function setTextarea(): void {
+    const para = document.querySelector("p");
+    const compStyles = window.getComputedStyle(para!);
+
+    const observer = new MutationObserver(() => {
+      setRoundCornersCss(compStyles.getPropertyValue("border-radius"));
+      setBackgroundColor(compStyles.getPropertyValue("background-color"));
+      setPadding(compStyles.getPropertyValue("padding"));
+    });
+
+    const target = document.getElementById("boxStyle");
+    const config = { attributes: true, childList: false, subtree: false };
+    observer.observe(target!, config);
+  }
+
+  function copyTextarea(): void {
+    const copyText = document.getElementById("text") as HTMLInputElement;
+    navigator.clipboard.writeText(copyText.value).then(() => alert("Copied the text: " + copyText.value));
+  }
```

Comment:
```suggestion
  const textAreaValue = `border-radius: ${upperLeftRadius}px ${upperRightRadius}px ${lowerLeftRadius}px ${lowerRightRadius}px`

  function copyTextarea(): void {
    navigator.clipboard.writeText(textAreaValue).then(() => alert("Copied the text: " + textAreaValue));
  }
```
こんな感じで TextAreaValue を一旦変数に入れておいて copyTextarea に渡したり textarea の value に指定するだけで良いかと